### DISCUSSION TITLE
How to tell weblate to ignore a commit?

### DISCUSSION THREAD

**Discussion by tttp** [WeblateOrg/weblate#9166] [Q&A]
so I made a mistake: 

I deleted a bunch of keys that shouldnt' have into my source languages and git pushed it

and I have set up weblate to remove keys into the translations when they are removed from the source

I did notice my mistake and added the keys into the source language and pushed it.

so I know have a PR from weblate that has all the deleted+restored keys that are machine translated in all the languages

and they are overwriting the translations that have been carefully corrected by humans, and if I accept weblate PR, I will replace the humans translations by machine translations.

for instance in nl, weblate + machine learning has reverted back to

"Another": "Een andere",

what was rightly manually corrected as 

 "Another": "Nog een",

what's the best way of correcting that? is there a way to force push the locales (non source) into weblate? revert the commit from hell that deleted the keys or remove the ones with automatic translations

@nijel suggested me https://docs.weblate.org/en/latest/faq.html#how-to-handle-renaming-translation-files
but my understanding is that it's to prevent the fuckup, not to correct it (or at least I don't get how)

Many thanks in advance

**Comment by nijel** [comment]
Push whatever you consider correct to the upsteam repository and hit "Reset" in Weblate.
